modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase

```diff
@@ -278,13 +278,165 @@ definition {
 			value1 = "Amortiguador");
 	}
 
-	@description = "This is a test for COMMERCE-5805. UseSearchBarAndPagination"
+	@description = "COMMERCE-5802. Use a filter always check that the results are correct"
 	@ignore = "true"
 	@priority = "3"
+	test UseFilters {
+		property portal.acceptance = "false";
+
+		CommerceNavigator.gotoPortlet(
+			category = "Store Management",
+			portlet = "Channels");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewChannelsEntry(
+			channelCurrency = "US Dollar",
+			channelName = "Test Channel",
+			channelType = "Site");
+
+		Button.clickSave();
```

Comment:
This step is already executed in the setup by the macro CommerceConfiguration#commerceStoreFrontSetUp

Reply:
In progress

Reply:
Done